### PR TITLE
Refactors User Controller

### PR DESCRIPTION
**Pull request by Devonte202**
- Deletes password from JWT
- Add comments
- Consolidates methods from User.js

### CODE REVIEW

File: controllers/userController.js [hunk 1]
```diff
@@ -12,20 +18,30 @@ const validateInputs = (username, email, password) => {
...
     const saltRounds = 7
     const hashedPassword = await bcrypt.hash(password, saltRounds)
-    User.addUser(username, email, hashedPassword)
-    const token = jwt.sign({ username: username, password: hashedPassword }, serverKey)
+    User.add(username, email, hashedPassword)
+    const token = jwt.sign({ username: username }, serverKey)
     res.cookie('safeToken', token)
   } catch (err) {
-    res.send('err')
+    res.status(401).send(err)
```
Comment (emtes) [inline controllers/userController.js:36]: Cool way to catch the error you throwed

File: controllers/userController.js [hunk 2]
```diff
@@ -3,6 +3,12 @@ const jwt = require('jsonwebtoken')
 const User = require('../models/User')
 const serverKey = require('./.keyEnv')
```
[the remark this answers is not in the document]
Comment (Devonte202) [reply controllers/userController.js:4]: Its a place holder file/route, but fundamentally an .env file

File: controllers/userController.js (hunk 1 above)
Comment (Devonte202) [reply controllers/userController.js:36]: Thanks! :D

File: controllers/userController.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Devonte202) [reply controllers/userController.js:4]: But I'll change it to a .env file

**Pull request merged**